server/src/models/BoardGame.js

```diff
@@ -12,32 +13,43 @@ class BoardGame extends unique(Model) {
   }
 
   static get relationMappings() {
-	const { Review } = require("./index")
-
-	return {
-		reviews: {
-			relation: Model.HasManyRelation,
-			modelClass: Review,
-			join: {
-				from: "boardGames.id",
-				to: "reviews.boardGameId"
-			}
+		const { Review } = require("./index")
+		const {User} = require("./index")
+
+
+		return {
+			reviews: {
+				relation: Model.HasManyRelation,
+				modelClass: Review,
+				join: {
+					from: "boardGames.id",
+					to: "reviews.boardGameId"
+				}
+			}, 
+			
+			user: {
+		relation: Model.BelongsToOneRelation,
+		modelClass: User,
+		join: {
+		  from: "boardGame.userId",
+		  to: "users.id"
+		}
+	  }
```

Comment:
fix your indentation - lines 31-37 should be indented, nested under the `user` relationMapping on line 30